app/README.md

```diff
@@ -210,4 +213,3 @@ You can expect the template engine to yield the following:
 ``` sh
 "BAR everything"
 ```
```

Comment:
Should the readme have instructions on local queue setup for development? (or is this somewhere else I've missed)?

Reply:
Oh crap you're right - that completely slipped my mind. I'll add that in shortly